packages/spanner/src/database.js

```diff
@@ -577,6 +593,25 @@ Database.prototype.run = function(query, options, callback) {
  *   .on('end', function() {});
  *
  * //-
+ * // If you need to enforce a specific param type, a types map can be provided.
+ * // This is typically useful if your param value can be null.
```

Comment:
Would you mind changing the example to show the null case, and recommend passing values in their native-type format, e.g. `spanner.int` instead of just the number?